src/domain/swap/index.ts

```diff
@@ -0,0 +1,24 @@
+export * from "./swap-out-checker"
+
+export const SwapProvider = {
+  Loop: "Loop",
+} as const
+
+export const SwapType = {
+  Swapout: "swapout",
+  Unknown: "unknown",
+} as const
+
+export const SwapState = {
+  Initiated: "initiated",
+  PreimageRevealed: "preimageRevealed",
+  HtlcPublished: "htlcPublished",
+  Success: "success",
+  Failed: "failed",
+  InvoiceSettled: "invoiceSettled",
+} as const
+
+export const LoopdInstanceName = {
```

Comment:
Definitely don't want couple `domain` to the specifics of our infra setup (with the hard coded lnd1/2 names)... at least I'm doubtful that this is right.

Reply:
removed this from the domain